pkg/tfvars/openstack/openstack.go

```diff
@@ -211,6 +233,7 @@ func TFVars(
 		MachinesNetwork                   string                            `json:"openstack_machines_network_id,omitempty"`
 		MasterAvailabilityZones           []string                          `json:"openstack_master_availability_zones,omitempty"`
 		MasterRootVolumeAvailabilityZones []string                          `json:"openstack_master_root_volume_availability_zones,omitempty"`
+		MasterPortsJSON                   []string                          `json:"openstack_master_ports_json,omitempty"`
```

Comment:
I think `omitempty` can be removed. This was suggested to me on another PR because `masterPortsJSON` is always going to be set and will never be empty.

Reply:
For sure it’s irrelevant.

Reply:
Removed!